core/lexicon/en/template.inc.php

```diff
@@ -33,7 +33,7 @@
 $_lang['template_err_remove'] = 'An error occurred while removing the template.';
 $_lang['template_err_save'] = 'An error occurred while saving the template.';
 $_lang['template_icon'] = 'Icon';
-$_lang['template_icon_description'] = 'Optional. A custom icon class for all resources with this template.';
+$_lang['template_icon_description'] = 'Optional. A custom Font Awesome (Free) icon class for all resources with this template. For example: icon-home or fa-home.';
```

Comment:
`css icon class` instead of `Font Awesome (free) icon class`?

Reply:
I am change.

Reply:
@Jako Without clarifying Font Awesome, it’s not obvious which class to use, especially for new users.

Reply:
@Jako I agree with Ruslan. This description was born during the discussion.

Reply:
Then write `(i.e. a Font Awesome class)`, since that part depends on the css of the manager.

Reply:
```suggestion
$_lang['template_icon_description'] = 'Optional. A custom CSS icon class for all resources with this template. In example a Font Awesome (free) class such as icon-home or fa-home.';
```